### ISSUE TITLE
Numpy array as output format

### ISSUE THREAD

**Issue by j-dominguez9** [elevenlabs/elevenlabs-python#316]
Maybe I'm missing something, but when I updated the package to the newest version, the output format changed to a generator object such that gradio no longer is able to process the output from client.generate() anymore. The examples don't help either as I don't want to save each output to a file and read it again and streaming option also doesn't work with gradio. I don't want to/cant use the `play` and `stream` functions.

**Comment by jayjcoding** [comment]
Yes, I am having the same issue. client.generate() does not work. A possible fix would be to use text_to_speech but the code will need to be modified.

**Comment by j-dominguez9** [comment]
@jayjcoding For reference, this is what I had to do to get it to work, but it's absurd and unnecessary that it had to go through all those steps:

from scipy.io import wavfile
from pydub import AudioSegment
from io import BytesIO

def tts_out(text: str):
    response = elevenlabs_client.text_to_speech.convert(
        text=text,
        voice_id="xxxxxxxxxxxxxxx",
        optimize_streaming_latency="0",
        output_format="mp3_22050_32",
    )

    audio_stream = BytesIO()

    for chunk in response:
        if chunk:
            audio_stream.write(chunk)

    audio_stream.seek(0)

    audio = AudioSegment.from_mp3(audio_stream)

    wav_io = BytesIO()
    wav_file = audio.export(wav_io, format="wav")
    wav_io.seek(0)

    sample_rate, audio_data = wavfile.read(wav_io)

    return (sample_rate, audio_data)

**Comment by dsinghvi** [comment]
@j-dominguez9 @jayjcoding thanks for the feedback. Would you be interested in making a contribution?

**Comment by jayjcoding** [comment]
Sure. I'd love to but I don't know a lot about this. If I can, then I would gladly do.

**Comment by j-dominguez9** [comment]
@dsinghvi Thanks for the opportunity. If you point me in the right direction to save time, like what files would be relevant to look at, and what you had in mind for changes, I'd be happy to help.

**Comment by dsinghvi** [comment]
@j-dominguez9 I could see you adding a new `for_gradio` helper function and adding the file in this [directory](https://github.com/elevenlabs/elevenlabs-python/tree/main/src/elevenlabs). Then users can import the function if they want to use `gradio` directly. 

You should be able to clone the repo like any Python SDK repository, run `poetry install` and ship up a PR!

**Comment by AngeloGiacco** [comment]
i'm closing this issue as it is stale, but i will work on getting this feature request shipped
